conmon-rs/server/src/rpc.rs

```diff
@@ -102,8 +103,13 @@ impl conmon::Server for Server {
             id, timeout,
         );
 
-        let console = pry_err!(Console::new());
-        let args = pry_err!(self.generate_exec_sync_args(&pidfile, Some(&console), &params));
+        let console_or_iostreams = if req.get_terminal() {
+            pry_err!(Console::new()).into()
+        } else {
+            pry_err!(IOStreams::new()).into()
+        };
```

Comment:
Could we have a constructor of console_or_iostreams that takes a bool and handles this opaquely? I don't see much of a need for the caller to know which it is

Reply:
Yep, done :+1: